### PR TITLE
Introduction to SQLi

### PR DESCRIPTION
**Pull request by anas-jamal**

### CODE REVIEW

**Comment by bitvijays** [comment]
@anas-jamal Hope you are doing well. Thank you for the pull request. Could just double check for the Markdown Linter problems? and have a look.

**Comment by anas-jamal** [comment]
Hey Vijay i have checked the mardown linter problems only duplicate names errors remaining and we can ignore them other problems are resolved lemme know if you face any issue.

File: docs/Series_Capture_The_Flag/LFC-WebExploitation.md [hunk 1]
```diff
@@ -992,19 +991,1371 @@ The problem here is PHPMailer does not clean the user supplied email values befo
...
...es in use on the server. 10. Review the names of all session tokens issued by the application to identify the technologies being used. 11. Use lists of common technologies, or Google, to establish which technologies may be in use on the server, or discover other websites and applications that appear to employ the same technologies. 12. Perform searches on Google for the names of any unusual cookies, scripts, HTTP headers, and the like that may belong to third-party software components. If you locate other applications in which the same components are being used, review these to identify any additional functionality and parameters that the components support, and verify whether these are also present in your target application. Note that third-party components may look and feel quite different in each implementation, due to branding customizations, but the core functionality, including script and parameter names, is often the same. If possible, download and install the component and analyze it to fully understand its capabilities and, if possible, discover any vulnerabilities. Consult repositories of known vulnerabilities to identify any known defects with the component in question.
```
Comment (bitvijays) [inline docs/Series_Capture_The_Flag/LFC-WebExploitation.md:2290]: Please correct this. Should be point wise

File: docs/Series_Capture_The_Flag/LFC-WebExploitation.md [hunk 2]
```diff
@@ -992,19 +991,1371 @@ The problem here is PHPMailer does not clean the user supplied email values befo
...
+
+2. Try to think like a programmer, and imagine what server-side mechanisms and technologies are likely to have been used to implement the behavior you can observe.
+
+HACK STEPS
+
+1. Try to identify any locations within the application that may contain clues about the internal structure and functionality of other areas.
+
+2. It may not be possible to draw any firm conclusions here; however, the cases identified may prove useful at a later stage of the attack when you’re attempting to exploit any potential vulnerabilities.
+
+HACK STEPS
```
Comment (bitvijays) [inline docs/Series_Capture_The_Flag/LFC-WebExploitation.md:2304]: Could you double-check these hack steps are in correct sequence? It seems currently, this has been just copied pasted from the previous one. We need to ensure it is in sync while reading

Comment (anas-jamal) [reply docs/Series_Capture_The_Flag/LFC-WebExploitation.md:2304]: Yes double checked from the book itself all hack steps are in correct sequence.

**Pull request merged**